rel/files/mongooseimctl

```diff
@@ -219,8 +239,8 @@ done
 
 case $1 in
     'start') start;;
-    'add_to_cluster') add_to_cluster;;
-    'remove_from_cluster') remove_from_cluster;;
+    'join_cluster') join_cluster;;
+    'leave_cluster') leave_cluster;;
```

Comment:
Command `join_cluster` takes as a parameter some cluster member and it's called from joining node ("the alone one")
Command 'leave_cluster' takes no parameters and it's called from  the node which is going to leave the cluster.
In your scenario, if we have clustered nodes A, B and C and we want to remove C we have to call `leave_cluster` from node C
What I've done in this PR was just a pure command line interface for this module [1] and maybe I didn't understand the task correctly?
[1]https://github.com/esl/MongooseIM/blob/20d40e87ab492013b77c32153fbce392890efae1/apps/ejabberd/src/mongoose_cluster.erl